web/themes/custom/cloudy/webpack.config.js

```diff
@@ -10,6 +10,7 @@ const config = {
     cloudy: './src/cloudy.scss',
     cloudyJs: './src/cloudy.js',
     'search-field': './src/js/search-field.js',
+    'back-to-top': './src/js/back-to-top.js',
```

Comment:
My code validator is showing this as invalid javascript without the quotes, due to the hyphens in the names. I'm going to leave them for now, rather than changing to camel case and removing the quotes, so that I don't break anything. When we do a standards audit, we can look at changing it to camel case, updating for all instances, and regression testing it.